bundle.yaml

```diff
@@ -14,10 +15,44 @@ applications:
     charm: grafana-k8s
     channel: edge
     scale: 1
+  loki:
+    charm: loki-k8s
+    channel: edge
+    scale: 1
 
 relations:
-- - prometheus
-  - alertmanager
-- - grafana
-  - prometheus
+- - prometheus:alertmanager
+  - alertmanager:alerting
+- - grafana:grafana-source
+  - prometheus:grafana-source
+#- - grafana:grafana-source
+#  - loki:grafana-source
+
+--- # overlay.yaml
+# Ref: https://discourse.charmhub.io/t/bundle-reference/1158
+applications:
```

Comment:
We also need:

* prometheus:metrics-endpoint https://github.com/canonical/prometheus-operator/blob/2d9a0137a73afe0cc6312b1040235ec4da0e384d/metadata.yaml#L29
* loki:logging https://github.com/canonical/loki-operator/blob/b594afe343cba3a44c7fffe0c2bb11063400470a/metadata.yaml#L39